documentation/docs/how_tos/outputs_and_transactions/06_check_unlock_conditions.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Check Unlock Conditions
+description: 'How to check unlock conditions of an output.'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- check unlock conditions
+- output
+- nodejs
+- python
+- java
+- rust
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import NodejsOutput from "../../libraries/nodejs/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+import PythonCode from "../../libraries/python/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import PythonOutput from "../../libraries/python/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+import JavaCode from "../../libraries/java/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import JavaOutput from "../../libraries/java/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+import RustCode from "../../libraries/rust/how_to/outputs_and_transactions/_06_check_unlock_conditions/_code_sample.mdx";
+import RustOutput from "../../libraries/rust/how_to/outputs_and_transactions/_06_check_unlock_conditions/_expected_output.md";
+
+Outputs may have multiple [UnlockConditions](https://github.com/iotaledger/tips/blob/main/tips/TIP-0018/tip-0018.md#unlock-conditions) which may require returning some or all of the amount, which could expire if not claimed in time, or which may not be unlockable for a very long time.
```

Comment:
Once we're happy with the text in the warning, we should update this as well with the first two sentences.

Reply:
Updated